### PR TITLE
[Mapfish] Add context filter by role doc

### PR DESCRIPTION
**Pull request by jusabatier**

### CODE REVIEW

**Comment by jusabatier** [comment]
Right, I corrected it

File: mapfishapp/app/js/GEOR_custom.js [hunk 1]
```diff
@@ -31,13 +31,17 @@ GEOR.custom = {
      *   * the keywords used to filter the view
      *
      * Should *not* be empty !
+     * 
+     * And an optional field : 
+     *   * the roles which can view the context
```
Comment (fvanderbiest) [inline mapfishapp/app/js/GEOR_custom.js]: TODO: specify what happens when empty, or missing. Thanks !

**Comment by fvanderbiest** [comment]
Squashing the commits here would be better too, thanks.

**Comment by jusabatier** [comment]
Done

**Pull request merged**